src/ios/GMImagePicker/GMGridViewController.m

```diff
@@ -532,8 +532,16 @@ - (BOOL)collectionView:(UICollectionView *)collectionView shouldSelectItemAtInde
                 fetch_item.be_progressed = false;
                 fetch_item.be_finished = true;
                 
-                AVURLAsset* urlAsset = (AVURLAsset*)asset;
-                fetch_item.image_fullsize = [urlAsset.URL absoluteString];
+                AVURLAsset *urlAsset = (AVURLAsset*)asset;
+                NSString *srcPath = [urlAsset.URL absoluteString];
+                NSString *videoExtension = [srcPath substringFromIndex:[srcPath length] - 3];
```

Comment:
Technically this will only work for 3 character file extensions... so like `.mpeg` wouldn't work. But if this works in practice for the different file formats you tested I'm okay with it.